.github/workflows/build.yml

```diff
@@ -440,4 +440,43 @@ jobs:
             zetacored-ubuntu-20-amd64
             zetaclientd-ubuntu-20-amd64
 
+      - name: Determine Release Type
+        run: |
+          if [[ "${{ env.BINARY_VERSION }}" =~ ^v[0-9]+\.0\.0+$ ]]; then
+            echo "RELEASE_TYPE=major" >> ${GITHUB_ENV}
+          elif [[ "${{ env.BINARY_VERSION }}" =~ ^v[0-9]+\.[0-9]+\.[1-9]+$ ]]; then
+            echo "RELEASE_TYPE=minor" >> ${GITHUB_ENV}
+          else
+            echo "RELEASE_TYPE=unknown" >> ${GITHUB_ENV}
+          fi 
+        id: determine_release_type
 
+      - name: "SEND:DISCORD:MESSAGE"
+        if: steps.determine_release_type.outputs.RELEASE_TYPE == 'major'
+        uses: gzukel/CosmosComposites/send_discord_message@main
+        with:
+          discord_token: "${{ secrets.DISCORD_TOKEN }}"
+          discord_channel_id: "${{ secrets.DISCORD_CHANNEL_ID }}"
+          discord_message: |
+            Hey <@&1122981184255840306>! A new version of the zetachain node has been released. 
```

Comment:
Change - `A new version of the zetachain node has been released` > `A new version of the ZetaChain node software has been released.`

ZetaChain should be capitalized at the Z and C in communications and ` ZetaChain software` feels more accurate to me than `node` since we aren't releasing a node itself.